Dockerfile

```diff
@@ -1,45 +1,20 @@
-FROM python:2.7
-LABEL maintainer "asv13@pitt.edu"
+FROM ubuntu
+LABEL maintainer "francois.lanusse@gmail.com"
 
 RUN apt-get update && apt-get upgrade -y
-RUN apt-get install -y git make g++ gcc wget swig libtool autoconf
-RUN pip install numpy ipython[all] scipy matplotlib
+RUN apt-get install -y git cmake make g++ gcc wget swig python-pip pkg-config
+RUN apt-get install -y libgsl-dev libfftw3-dev
+RUN pip install numpy scipy matplotlib jupyter
 
-ENV GSL_TAR="gsl-2.3.tar.gz"
-ENV GSL_DL="http://ftp.wayne.edu/gnu/gsl/$GSL_TAR"
-ENV FFTW_TAR="fftw-3.3.6-pl2.tar.gz"
-ENV FFTW_DL="http://www.fftw.org/$FFTW_TAR"
+# Installing CCL C library
+RUN git clone https://github.com/EiffL/CCL && cd CCL && \

```

Comment:
Once merged you'll probably want to change this and point to the LSSTDESC/CCL repo, right?

Reply:
Yes :-) It's there just so that I could build and test the Docker container before the merge